util/importers/fisher.py

```diff
@@ -0,0 +1,313 @@
+            
+            # Open wav corresponding to transcription file
+            wav_filename = os.path.splitext(os.path.basename(trans_file))[0] + ".wav"
+            wav_file = os.path.join(source_dir, wav_filename)
+            
+            print("splitting {} according to {}".format(wav_file, trans_file))
+            
+            origAudio = wave.open(wav_file, "r")
+            
+            # Loop over segments and split wav_file for each segment
+            for segment in segments:
+                # Create wav segment filename
+                start_time = segment["start_time"]
+                stop_time = segment["stop_time"]
+                new_wav_filename = os.path.splitext(os.path.basename(trans_file))[0] + "-" + str(start_time) + "-" + str(stop_time) + ".wav"
+                new_wav_file = os.path.join(target_dir, new_wav_filename)
+                
+                # If the wav segment filename does not exist create it
+                if not os.path.exists(new_wav_file):
+                    _split_wav(origAudio, start_time, stop_time, new_wav_file)
+            
+            # Close origAudio
+            origAudio.close()
+            
+            # Remove wav_file
+            # os.remove(wav_file)
+
+def _split_wav(origAudio, start_time, stop_time, new_wav_file):
+    frameRate = origAudio.getframerate()
+    origAudio.setpos(int(start_time*frameRate))
+    chunkData = origAudio.readframes(int((stop_time - start_time)*frameRate))
+    chunkAudio = wave.open(new_wav_file, "w")
+    chunkAudio.setnchannels(origAudio.getnchannels())
+    chunkAudio.setsampwidth(origAudio.getsampwidth())
+    chunkAudio.setframerate(frameRate)
+    chunkAudio.writeframes(chunkData)
+    chunkAudio.close()
+
+def _maybe_split_transcriptions(data_dir, original_data, converted_data):
+    source_dir = os.path.join(data_dir, original_data)
+    target_dir = os.path.join(data_dir, converted_data)
+    
+    if os.path.exists(os.path.join(source_dir, "split_transcriptions_done")):
+        print("skipping maybe_split_transcriptions")
+        return
+    
+    # Loop over transcription files and split them into individual files for
+    # each utterance
+    for root, dirnames, filenames in os.walk(source_dir):
+        for filename in fnmatch.filter(filenames, "*.txt"):
+            trans_file = os.path.join(root, filename)
+            segments = _parse_transcriptions(trans_file)
+            
+            # Loop over segments and split wav_file for each segment
+            for segment in segments:
+                start_time = segment["start_time"]
+                stop_time = segment["stop_time"]
+                txt_filename = os.path.splitext(os.path.basename(trans_file))[0] + "-" + str(start_time) + "-" + str(stop_time) + ".txt"
+                txt_file = os.path.join(target_dir, txt_filename)
+                
+                transcript = validate_label(segment["transcript"])
```

Comment:
@reuben @kdavis-mozilla If we are leaving out everything with '[' or ']', how do we handle situations like these (that happens on swb) for example:

`sw2001A-ms98-a-0019 76.424125 80.037375 yeah yeah well i[t]- i[t]- that's right and it`

`sw2001A-ms98-a-0029 153.333375 162.042250 the men often have to wear shirt and [laughter-tie] [laughter-no] [laughter-matter] r[ight]- right right what time of what time of the year that's right`

`sw2001A-ms98-a-0048 239.418375 243.819750 yeah [laughter] i don't know what el[se]- what more we can say about that that's`

 `sw2001A-ms98-a-0008 33.898250 40.123250 right right is there is there um an[y]- is there a like a code of dress where you work do they ask`